modules/tinymce/src/plugins/code/demo/ts/demo/Demo.ts

```diff
@@ -1,12 +1,126 @@
+import { Dialog } from '@ephox/bridge';
+import { Id } from '@ephox/katamari';
+
 import { TinyMCE } from 'tinymce/core/api/PublicApi';
 
 declare let tinymce: TinyMCE;
 
+interface Data {
+  search: string;
+}
+
 tinymce.init({
   selector: 'textarea.tinymce',
   plugins: 'code',
-  toolbar: 'code',
-  height: 600
+  toolbar: 'code tree',
+  height: 600,
+  setup: (ed) => {
```

Comment:
This demo is within 'code' plugin codebase. Maybe better to move it to Silver theme demos

Reply:
Correct, I just moved it to its own demo page in the silver demos folder